diff --git a/node_modules/expo-modules-core/android/cmake/main.cmake b/node_modules/expo-modules-core/android/cmake/main.cmake
index 7e584e1..a4421af 100644
--- a/node_modules/expo-modules-core/android/cmake/main.cmake
+++ b/node_modules/expo-modules-core/android/cmake/main.cmake
@@ -61,5 +61,6 @@ target_link_libraries(
   expo-modules-core
   PRIVATE
+  c++_shared
   ${LOG_LIB}
   android
   ${JSEXECUTOR_LIB}
